platform/ui/src/components/downloadDialog/DownloadDialog.js

```diff
@@ -75,24 +76,31 @@ const DownloadDialog = ({
   }, [defaultSize, disableViewport, enableViewport, viewportElement]);
 
   useEffect(() => {
+    const getValidMeasure = value => {
+      const min = minimumSize - 1;
+      return value > min && value < maximumSize
+        ? value
+        : value < min
+        ? minimumSize
+        : maximumSize;
```

Comment:
Let's avoid a nested ternary if we can help it

Reply:
Sure! thx, I have simplified this function because of the clamp for max values in the onChange event.